### DISCUSSION TITLE
can‘t work with SD_MMC

### DISCUSSION THREAD

**Discussion by liu2548253579** [schreibfaul1/ESP32-audioI2S#365] [Q&A]
I used the way of SD_MMC to read the sdcard content and then used "audio.connecttoFS(SD_MMC,"/music/paomo.mp3");" to link,but it didn't worked. I have verified the SD_MMC is available before that ，the error info from serial is in the first picture ，code is in the second picture，i would appreciate your reply
![image](https://user-images.githubusercontent.com/42726841/188085885-a6cc8bf4-5dc7-404c-ab33-aebd2405ced6.png)
![image](https://user-images.githubusercontent.com/42726841/188086623-0377fe01-e42b-4ece-b2a6-a9f025345394.png)

**Comment by schreibfaul1** [comment]
0x109 is ESP_ERR_INVALID_CRC, and send_scr is the first transaction which uses D0 line (all the previous ones use only CMD and CLK). 

set before SD_MMC.begin(...
````pinMode(SD_PIN_D0, INPUT_PULLUP);````

**Reply by liu2548253579** [reply]
Thanks for your swift reply,i tried and the errors still exit.
